load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -191,7 +204,8 @@ private void run() {
       Client.numberOfMessages = numberOfMessages;
       GCEController gceController = GCEController.newGCEController(
           project, ImmutableMap.of("us-central1-a", clientParamsMap),
-          Executors.newScheduledThreadPool(500));
+          Executors.newScheduledThreadPool(500), cpsPublisherCount,
```

Comment:
The clientParamsMap already has all these counts. It makes more sense to just iterate over the map to build your spreadsheet.